### PR TITLE
Just lol

### PR DESCRIPTION
**Pull request by mark-i-m**

### CODE REVIEW

File: src/appendix/humorust.md [hunk 1]
```diff
@@ -10,3 +10,4 @@ enlightening?
 - [Rust Koans](https://users.rust-lang.org/t/rust-koans/2408/4)
 - [`break rust;`](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=0ab2bd6a9d722e0f05a95e2a5dcf89cc)
 - [The Nomicon Intro](https://doc.rust-lang.org/stable/nomicon/)
+- [`rustc-ty` renaming punfest](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/rustc-ty.20naming.20bikeshed.20.2F.20punfest.20(was.3A.20design.20meeting.202.2E.2E.2E)
```
Comment (JohnTitor) [inline src/appendix/humorust.md]:
```suggestion
- [`rustc-ty` renaming punfest](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/rustc-ty.20naming.20bikeshed.20.2F.20punfest.20(was.3A.20design.20meeting.202.2E.2E.2E )
```

Hmm, markdown isn't laughing :/

**Comment by mark-i-m** [comment]
Perhaps I need to escape it somehow

**Comment by JohnTitor** [comment]
The change looks good! CI failures are unrelated, addressed by #604.

**Comment by JohnTitor** [comment]
Merged that PR, could you rebase?

**Comment by spastorino** [comment]
It was fun to hear @pnkfelix on one of the last meetings talking about some ICE-breakers groups names like [window-breakers](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/weekly.20meeting.202020-03-05.20.2354818/near/189804864) and [ARM breakers](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/weekly.20meeting.202020-03-05.20.2354818/near/189804501)

**Pull request merged**